planning/behavior_path_planner/src/scene_module/avoidance/avoidance_module.cpp

```diff
@@ -191,11 +197,14 @@ ObjectDataArray AvoidanceModule::calcAvoidanceTargetObjects(
   debug_linestring.clear();
   // for filtered objects
   ObjectDataArray target_objects;
+  AvoidanceInfoArray avoidance_info_array;
+  std::vector<std::string> avoidance_factors;
   for (const auto & i : lane_filtered_objects_index) {
     const auto & object = objects_candidate.objects.at(i);
     const auto & object_pos = object.kinematics.initial_pose_with_covariance.pose.position;
 
     if (!isTargetObjectType(object)) {
+      avoidance_factors.emplace_back(AvoidanceFactor::OBJECT_IS_NOT_TYPE);
       DEBUG_PRINT("Ignore object: (isTargetObjectType is false)");
```

Comment:
You can remove these debug_prints.

Reply:
Alright, the `DEBUG_PRINT` has been removed with the new commit.